From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: unknown <tim.kuechler@online.de>
Date: Sun, 8 Nov 2020 10:38:57 +0100
Subject: [PATCH] knockback bugfix


diff --git a/src/main/java/net/minecraft/server/EntityLiving.java b/src/main/java/net/minecraft/server/EntityLiving.java
index d417878a1584c9e5ec33b94fc65b29f84fb3a5e9..50a82255bd5ce2d0ce5364aeb0201a8b0c662292 100644
--- a/src/main/java/net/minecraft/server/EntityLiving.java
+++ b/src/main/java/net/minecraft/server/EntityLiving.java
@@ -1236,7 +1236,7 @@ public abstract class EntityLiving extends Entity {
                     this.velocityChanged();
                 }
 
-                if (entity1 != null) {
+                if (entity1 != null && (entity1 instanceof EntityHuman || entity1 instanceof EntityArrow)) {
                     double d0 = entity1.locX() - this.locX();
 
                     double d1;
