From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: unknown <tim.kuechler@online.de>
Date: Sun, 8 Nov 2020 10:48:08 +0100
Subject: [PATCH] knockback fix2


diff --git a/src/main/java/net/minecraft/server/EntityLiving.java b/src/main/java/net/minecraft/server/EntityLiving.java
index 50a82255bd5ce2d0ce5364aeb0201a8b0c662292..e1470439389d2020458ce76418cb3170bd102873 100644
--- a/src/main/java/net/minecraft/server/EntityLiving.java
+++ b/src/main/java/net/minecraft/server/EntityLiving.java
@@ -1236,7 +1236,7 @@ public abstract class EntityLiving extends Entity {
                     this.velocityChanged();
                 }
 
-                if (entity1 != null && (entity1 instanceof EntityHuman || entity1 instanceof EntityArrow)) {
+                if (entity1 != null && !(entity1 instanceof EntityHuman) && !(entity1 instanceof EntityArrow)) {
                     double d0 = entity1.locX() - this.locX();
 
                     double d1;
