From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Tim=20K=C3=BCchler?= <tim.kuechler@online.de>
Date: Sun, 24 Jan 2021 06:29:10 +0100
Subject: [PATCH] Deactivate Knockback Resistance Attribute


diff --git a/src/main/java/net/minecraft/server/EntityLiving.java b/src/main/java/net/minecraft/server/EntityLiving.java
index e1470439389d2020458ce76418cb3170bd102873..9026aebcb693eea50c126c731e4dcfa711cfe330 100644
--- a/src/main/java/net/minecraft/server/EntityLiving.java
+++ b/src/main/java/net/minecraft/server/EntityLiving.java
@@ -1564,7 +1564,6 @@ public abstract class EntityLiving extends Entity {
     }
     public void doKnockback(float f, double d0, double d1, Entity knockingBackEntity) {
         // Paper end - add knockbacking entity parameter
-        f = (float) ((double) f * (1.0D - this.b(GenericAttributes.KNOCKBACK_RESISTANCE)));
         if (f > 0.0F) {
             this.impulse = true;
             Vec3D vec3d = this.getMot();
